libvast/include/vast/system/actors.hpp

```diff
@@ -507,6 +507,9 @@ using node_actor = typed_actor_fwd<
   // TODO: Make this a signal_monitor_client_actor
   caf::reacts_to<atom::signal, int>>::unwrap;
 
+using terminator_actor = typed_actor_fwd<caf::replies_to<
+  atom::shutdown, std::vector<caf::actor>>::with<atom::done>>::unwrap;
```

Comment:
Throughout the file we force line breaks in the typed actor definitions like this by commenting every handler. This makes it easy to understand what a given handler does from looking at this one file only.

```suggestion
using terminator_actor = typed_actor_fwd<
  // Shut down the given actors.
  caf::replies_to<atom::shutdown, std::vector<caf::actor>>::with<
    atom::done>>::unwrap;
```